content/documentation/pages/5-feature-guides/1-streams/7-scaling.md

```diff
@@ -0,0 +1,33 @@
+---
+path: 'feature-guides/streams/scaling/'
+title: 'Scaling'
+description: 'Scaling streaming data pipeline with Spring Cloud Data Flow'
+---
+
+# Scaling
+
+Let's start with a general definition of scalability, the types, and why it is useful.
+
+[Scalability](https://en.wikipedia.org/wiki/Scalability) is the ability of a system to cope with increased load by adding resources. In Distributed Systems the ability to add resources such as CPUs, memory and storage to the existing nodes is known as [vertical scaling](https://en.wikipedia.org/wiki/Scalability#Vertical) (up/down), while [horizontal scaling](https://en.wikipedia.org/wiki/Scalability#Horizontal) (out/in) refers to the ability of adding additional nodes to the system.
+Following paragraphs discuss the common concepts for applying `horizontal scaling` (e.g. scaling out or shared nothing) to streaming data pipelines.
+
+[Stream processing](%currentPath%/concepts/streams/) in Spring Cloud Data Flow is implemented architecturally as a collection of independent event-driven streaming applications that connect using a messaging middleware of choice, for example RabbitMQ or Apache Kafka. The collection of independent applications come together at runtime to constitute a streaming data pipeline.
+The ability of a streaming pipeline to cope with increased data load depends on the following characteristics:
```

Comment:
"data pipeline" whenever possible for consistency (folks get it confused with CI pipeline)